### PR TITLE
LM Web Portfolio V2

### PR DESCRIPTION
**Pull request by LucasMontecino**
In this PR - Here's my second version of my web portfolio. In this case I added some new features and some css approach that I had the time to learned in the last two weeks. Also I added transitions, animations, transforms to the website. Finally, I added a scroll to top button navigation for mobile devices. I hope these new features looks good. 

P/D: In this case, I use a wrapper class to set my data-theme attribute to the page, and to wrap the elements I used a container (I hope this was a good implementation).

### CODE REVIEW

File: homeworks/05-css-advanced/styles.css [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+
+.container {
+    min-height: 100vh;
+    max-width: 1300px;
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 2rem 3rem;
+    gap: 2rem;
```
Comment (ababenko-solvd) [inline homeworks/05-css-advanced/styles.css:56]: In this case better not to set generic gap, but to adjust distance between sections pointly according to design

File: homeworks/05-css-advanced/styles.css [hunk 2]
```diff
@@ -0,0 +1,451 @@
...
+
+.section__secondary {
+    display: flex;
+    justify-content: center;
+    gap: 2rem;
+}
+
+.section__button {
+    width: 18.125rem;
+    padding: 1.438rem 0;
```
Comment (ababenko-solvd) [inline homeworks/05-css-advanced/styles.css:191]: You use too many 'rem'. Sizes and distances have to be just according to design - they are in pixels there!

File: homeworks/05-css-advanced/index.html [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+            </main>
+
+
+            <div class="scroll">
+                <button class="scroll__button">Go to Top!</button>
```
Comment (ababenko-solvd) [inline homeworks/05-css-advanced/index.html:281]: This button should be fixed positioned.

**Review by ababenko-solvd — APPROVED** [review]

File: homeworks/05-css-advanced/index.html (hunk 3 above)
Comment (LucasMontecino) [reply homeworks/05-css-advanced/index.html:281]: Ok, I will fix my mistakes ASAP

**Pull request merged**